README.md

```diff
@@ -1 +1,34 @@
-# hello-microverse
```

Comment:
[OPTIONAL] Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) as asked in the requirements.

- the name of the branch can be "add-homepage" for example.
- the PR must has a descriptive summary : 

<img width="604" alt="image" src="https://user-images.githubusercontent.com/7243132/170138148-cb0dddb9-af35-4899-930c-4cdc98367542.png">

the is an example to improve your : 
<img width="768" alt="image" src="https://user-images.githubusercontent.com/7243132/170138460-11d5404d-cb6b-47f9-84d1-742ddc8b3e9f.png">